src/main/java/ro/utcn/sd/mdantonio/StackUnderflow/entities/Post.java

```diff
@@ -0,0 +1,48 @@
+package ro.utcn.sd.mdantonio.StackUnderflow.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name="post")
+public class Post {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer postid;
+    private Integer posttypeid;
+    private Integer authorid;
+    private Integer parentid;
+    private String title;
+    private String body;
+
+    @Column(name="creationdate")
+    @Temporal(TemporalType.DATE)
+    private Date creationDate;
```

Comment:
I would strongly suggest that you use Java 8 time classes (e.g. LocalDate). The java.util.Date is older and not really used that much anymore (because of its flaws).